src/components/WebexMeetingControl/WebexMeetingControls.test.js

```diff
@@ -13,4 +13,15 @@ describe('Webex Meeting Controls component', () => {
 
     expect(component).toMatchSnapshot();
   });
+
+  test('matches snapshot with custom class', () => {
+    const meetingID = 'my-meeting';
+    const component = shallow(
+      <WebexMeetingControls className="my-custom-class" meetingID={meetingID}>
```

Comment:
I know you just copy-paste'd but just making an observation that for some reason we create a variable for the meeting ID on these tests, unlike hardcoding the ID directly like everywhere else